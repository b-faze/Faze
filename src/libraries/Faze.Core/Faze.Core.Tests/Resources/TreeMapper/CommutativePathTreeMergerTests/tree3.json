{
  "Value": "",
  "Children": [
    {
      "Value": "",
      "Children": [
        {
          "Value": "",
          "Children": [
            {
              "Value": "1"
            },
            {
              "Value": "2"
            }
          ]
        },
        {
          "Value": "",
          "Children": [
            {
              "Value": "3"
            },
            {
              "Value": "4"
            }
          ]
        }
      ]
    },
    {
      "Value": "",
      "Children": [
        {
          "Value": "",
          "Children": [
            {
              "Value": "5"
            },
            {
              "Value": "6"
            }
          ]
        },
        {
          "Value": "",
          "Children": [
            {
              "Value": "7"
            },
            {
              "Value": "8"
            }
          ]
        }
      ]
    }
  ]
}