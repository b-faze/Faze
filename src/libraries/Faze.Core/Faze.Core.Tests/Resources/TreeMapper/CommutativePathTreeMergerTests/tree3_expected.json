{
  "Value": "",
  "Children": [
    {
      "Value": "",
      "Children": [
        {
          "Value": "",
          "Children": [
            {
              "Value": "1"
            },
            {
              "Value": "2"
            }
          ]
        },
        {
          "Value": "",
          "Children": [
            {
              "Value": "2"
            },
            {
              "Value": "4"
            }
          ]
        }
      ]
    },
    {
      "Value": "",
      "Children": [
        {
          "Value": "",
          "Children": [
            {
              "Value": "2"
            },
            {
              "Value": "4"
            }
          ]
        },
        {
          "Value": "",
          "Children": [
            {
              "Value": "4"
            },
            {
              "Value": "8"
            }
          ]
        }
      ]
    }
  ]
}